src/components/charts/area-chart.scss

```diff
@@ -0,0 +1,10 @@
+.area-chart__y-tick-label {
```

Comment:
I know SCSS modules are on the horizon, in that case you can remove the `.area-chart__` stuff.